exporter/metric/metric.go

```diff
@@ -148,6 +148,10 @@ func (me *metricExporter) exportMetricDescriptor(ctx context.Context, rm metricd
 		}
 	}
 
+	if me.o.disableCreateMetricDescriptors {
```

Comment:
No reason to into this function at all if this option exists. It should either be at the top of this function, or in `Export` before this even gets called (probably the latter).

Reply:
Changed in: [33257b3](https://github.com/avilevy18/opentelemetry-operations-go/pull/1/commits/33257b31e3a9cec0824a4a00369e9f7a587ab8a8)